.travis.yml

```diff
@@ -3,10 +3,18 @@ python:
   - 3.4
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
```

Comment:
Sources for Travis apt addon are managed in a [whitelist](https://github.com/travis-ci/apt-source-whitelist)

PPA referenced in the [Ubuntu docs](https://wiki.ubuntu.com/ToolChain). Afaict it's the only source for g++4.8 for Ubuntu 12.04.